src/logic/operate.js

```diff
@@ -0,0 +1,18 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const oneB = Big(numberOne);
+  const twoB = Big(numberTwo);
+
+  const aritmethic = {
+    addition: oneB.plus(twoB),
+    substraction: oneB.minus(twoB),
+    multiplication: oneB.times(twoB),
+    division: oneB.div(twoB),
+    percentage: (oneB * twoB) / 100,
```

Comment:
Let's make use of `Modifier`.
![image](https://user-images.githubusercontent.com/31897434/127428519-a467d545-2c9f-4f01-8d8f-5ae5f2fc4823.png)


Example:
`oneB.mod(twoB)`